gui-win32/screen.c

```diff
@@ -192,12 +194,12 @@ winproc(void *a)
 	wc.lpszMenuName = 0;
 	wc.lpszClassName = L"9pmgraphics";
 	RegisterClass(&wc);
-
+    windowStyle = WS_OVERLAPPEDWINDOW;
```

Comment:
odd... this shows up as in-line on my local machine. Ditto for the case statement below. I'll take a look when I get home tonight; perhaps a tab character or something has slipped in there.